data/stack/eessi.yaml

```diff
@@ -0,0 +1,42 @@
+jupyterhub::kernel::setup: venv
+jupyterhub::kernel::venv::python: /cvmfs/pilot.eessi-hpc.org/2020.10/software/x86_64/intel/skylake_avx512/software/Python/3.8.2-GCCcore-9.3.0/bin/python
```

Comment:
This is not architecture proof, maybe we need a wrapper for this in EESSI? Would this be possible @boegel?